src/hooks/useWallets.js

```diff
@@ -1,24 +1,38 @@
+import { isEmpty } from 'lodash';
 import { useSelector } from 'react-redux';
+import WalletBackupTypes from '../helpers/walletBackupTypes';
 import WalletTypes from '../helpers/walletTypes';
 
 export default function useWallets() {
-  const {
-    isCreatingAccount,
-    selectedWallet,
-    walletNames,
-    wallets,
-  } = useSelector(
-    ({ wallets: { isCreatingAccount, selected, walletNames, wallets } }) => ({
-      isCreatingAccount,
+  const { isWalletLoading, selectedWallet, walletNames, wallets } = useSelector(
+    ({ wallets: { isWalletLoading, selected, walletNames, wallets } }) => ({
+      isWalletLoading,
       selectedWallet: selected || {},
       walletNames,
       wallets,
     })
   );
 
+  let latestBackup = false;
```

Comment:
I referred to this in my other comment in redux/wallets.js to have a separate "getLatestBackup" function that can be shared across both places - but just also wanted to note that this can be memo'd (or the results of that new separate function can be memo'd)